usaspending_api/common/elasticsearch/client.py

```diff
@@ -77,6 +77,29 @@ def es_client_query(
     return None
 
 
+def es_client_query_count(
```

Comment:
There is a lot of duplicated code being added in this PR.

Perhaps the cleanest solution would be to move most of these functions and the ES client into a new base class.
Then, child classes can implement the minor differences between the search vs count DSL functions

@kwhickey @BrianZito for additional pov

Reply:
I've been keeping my head out of ES code for a while, but have we not done this stuff before? It seems like waiting on an ES client would be nessicary for ANY Elasticsearch functionality.

Reply:
Using aggregation negated the need to hit the count endpoint, so duplicate code was removed